test/Utils.purs

```diff
@@ -67,9 +67,7 @@ interpret = interpretWithConfig defaultConfig { timeout = Just (wrap 50000.0) }
 interpretWithTimeout
   :: Maybe Milliseconds -> TestPlanM (Aff Unit) Unit -> Aff Unit
 interpretWithTimeout timeout spif = do
-  plan <- planT spif
-  runSpec' defaultConfig { timeout = timeout } [ consoleReporter ] $
-    planToSpec plan
+  interpretWithConfig (defaultConfig { timeout = timeout }) spif
```

Comment:
Made this change while I was beginning to use `runSpecT` instead of `runSpec'`, but might as well keep it